Mathlib/Order/SuccPred/WithBot.lean

```diff
@@ -39,6 +39,21 @@ lemma succ_strictMono [NoMaxOrder α] : StrictMono (succ : WithBot α → α)
 @[gcongr] lemma succ_le_succ (hxy : x ≤ y) : x.succ ≤ y.succ := succ_mono hxy
 @[gcongr] lemma succ_lt_succ [NoMaxOrder α] (hxy : x < y) : x.succ < y.succ := succ_strictMono hxy
 
+section LinearOrder
+
+variable {α : Type*} [Nontrivial α] [LinearOrder α] [OrderBot α] [SuccOrder α]
+
+@[simp]
+theorem succ_zero_iff (a : WithBot α) : WithBot.succ a = ⊥ ↔ a = ⊥ := by
```

Comment:
```suggestion
theorem succ_eq_bot {a : WithBot α} : WithBot.succ a = ⊥ ↔ a = ⊥ := by
```
I think you should add another simp lemma `succ_eq_zero {a : WithBot α} : WithBot.succ a = 0 ↔ a = ⊥` in `Algebra.Order.SuccPred`